scripts/dev-restart.sh

```
@@ -0,0 +1,172 @@
+# Menghentikan semua proses Node.js dan menjalankan ulang dengan port yang benar
+
+echo "🔄 Restarting Zapin SaaS Development Servers..."
+echo "================================================"
+
+# Function untuk membunuh proses di port tertentu
+kill_port() {
+    local port=$1
+    echo "🔍 Checking port $port..."
+    
+    # Cari PID yang menggunakan port
+    local pid=$(lsof -ti:$port)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)